MediaPlayer/MediaControls.js

```diff
@@ -209,70 +271,16 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {	// eslint-disable-line
 	class MediaControlsDecoratorHOC extends React.Component {
```

Comment:
Now `MediaControlsDecorator` do nothing. Let's remove it.

Then we could update the following code.
```
// const MediaControls = MediaControlsDecorator(MediaControlsBase);

export default MediaControlsBase;
export {
	MediaControlsBase as MediaControls,
	MediaControlsBase,
	MediaControlsDecorator
};
```

Reply:
alright